content/en/docs/tasks/security/automtls/index.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Automatic mutual TLS
+description: A simplified workflow to adopt mutual TLS with minimal configuration overhead.
+weight: 10
+keywords: [security,mtls,ux]
+---
+
+This tasks shows a simplified workflow for mutual TLS adoption[authentication overview](/docs/concepts/security/#authentication).
+
+With Istio auto mutual TLS feature, you can adopt mutual TLS by only configuring Authentication Policy without worrying about destination rule.
+
+Istio tracks the server workloads are migrated to Istio sidecar, and configure client sidecar to send mutual TLS traffic to those workloads automatically, and send plain text traffic to workloads
+without sidecars. This allows you to adopt Istio mutual TLS incrementally with minimal manual configuration.
+
+## Before you begin
+
+* Understand Istio [authentication policy](/docs/concepts/security/#authentication-policies) and related
+[mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) concepts.
+
+* Have a Kubernetes cluster with Istio installed (e.g use `install/kubernetes/istio-demo.yaml` as described in
+[installation steps](/docs/setup/install/kubernetes/#installation-steps), set `global.mtls.enabled` to false  and `global.mtls.auto` to true using [Helm](/docs/setup/install/helm/)). For example,
+
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system  \
```

Comment:
Helm is deprecated. Need to use istioctl manifest.

Reply:
done.